data_models/atlassian_data_model.py

```diff
@@ -0,0 +1,24 @@
+import panther_event_type_helpers as event_type
+from panther_base_helpers import deep_get
+
+ auditLogTypeMap = {
```

Comment:
nice!

Reply:
Gotta hand credit to @darwayne , this was his idea and a damn good one.